### ISSUE TITLE
Auto download from avistaz rss list not working

### ISSUE THREAD

**Issue by Buster14** [Radarr/Radarr#1935]
**Radarr Version: 0.2.0.778*

**Logs:**

Hi, i'm trying to get radarr working with avistaz's rss list.
What i want is everytime a movie that contains Bluray 720p released in avistaz, radarr automatically grabs it.
But it doesn't seem to be working, 
here's the setup pic and the rss list

http://imgur.com/625JjNN
https://drive.google.com/file/d/0Bwc1KPPVNa_9V1BlN2c5ck50cGc/view?usp=sharing
I don't want the new movies to automatically added to my collection because my library is alphabetic-based. 
The folder seems to be gone every time i save it, not sure if this is a bug or not..
So in short i want some kind of auto downloader, i cant figure out what im missing here...
Also, I'm not sure which part of radarr logs contain this, so please inform me what should i search for in the logs files.

**Comment by galli-leo** [comment]
This won't be possible with Radarr, since you first need to add the movies, before they are downloaded.
